nur/index.py

```diff
@@ -42,7 +44,43 @@ def index_repo(directory: Path, repo: str, expression_file: str) -> Dict[str, An
             if position is not None and position.startswith("/nix/store"):
                 path_str, line = position.rsplit(":", 1)
                 path = Path(path_str)
+                # I've decided to just take these 2 repositories,
```

Comment:
This function gets too long. Could you move the loop body to a different function that returns `pkg` which is than assigned to pkgs.